packages/shorebird_cli/test/src/commands/patch/patch_ios_command_test.dart

```diff
@@ -474,6 +475,42 @@ base_url: $baseUrl''',
       );
     });
 
+    test('provides appropriate ExportOptions.plist to build ipa command',
```

Comment:
ideally we shouldn't need to duplicate the test :(

Reply:
Yep, we'll get that once we write a ShorebirdBuildWrapper or similar and move this code out of a mixin :)